services/download/src/main/resources/application.yaml

```diff
@@ -14,12 +14,13 @@ logging:
 
 services:
   download:
+    target-system: EFGS
```

Comment:
I would think that having default target system is not ideal. I think we should enforce that a specific connect-xyz profile is always specified and fail fasst otherwise. What do you think?

Reply:
ok